pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -324,6 +324,12 @@
     )
     private long topicLoadTimeoutSeconds = 60;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            doc = "Configuration file path for local metadata store. It's supported by RocksdbMetadataStore for now."
+    )
+    private String metadataStoreConfigPath = null;
```

Comment:
```suggestion
    private String rocksDbMetadataStorePath = null;
```

WDYT @merlimat

Reply:
@codelipenghui Do you mean rocksDbMetadataStoreConfigPath? This is used for configs. Data store path is defined by metadataURL.